OrbitClientProtos/capture_data.proto

```diff
@@ -92,7 +92,7 @@ message TimerInfo {
   Type type = 6;
 
   int32 processor = 7;
-  uint64 callstack_hash = 8;
+  uint64 callstack_id = 8;
```

Comment:
this field is now unused - I do not know if we should remove it or keep it for the future?

Reply:
I would leave this question for @pierricgimmig : he wanted to leave this field and use it in near future

Reply:
So probably just keep it for now

Reply:
This would be used for callstacks of dynamically instrumented functions and scheduling slices I assume.